README.md

```diff
@@ -21,6 +23,25 @@ Open the overlay and put it on either the top left or top right of your screen.
 - Supports Windows and macOS (macOS has few visual bugs)
 - Completely FREE!
 
+## Upcoming features
+- [ ] Linux support
+- [ ] Fullscreen (Will take awhile)
+- [ ] Manually add api key
+- [ ] Search for player stats
+- [ ] Improve sniper detection using login dates and other stats
+- [ ] Show guilds
+- [ ] Remember overlay position and size
+- [ ] Custom file path
+- [ ] Hover over names to see all stats
+- [ ] Game start timer
```

Comment:
Added upcoming features to make the overlay look more appealing and to show there is development